### ISSUE TITLE
Duplicated Items/Lines - Concurrency API Issue 

### ISSUE THREAD

**Issue by Wolf2323** [DecentSoftware-eu/DecentHolograms#98]
### Just making sure

- [X] I made sure, this bug is not already known or reported.
- [X] I believe, that this bug is fixable.

### Reproduction

The issue is that Items/Lines are shown double. I think this video shows it the best what the issue is, when my plugin gets reloaded: https://cdn.discordapp.com/attachments/520942709205237760/1055568271723737189/Minecraft_1.19.3_-_Multiplayer_3rd-party_Server_2022-12-22_20-28-32.mp4

You can go two ways to reproduce this issue,
1. Build a setup that first disable the hologram via `.disable()` and then call the `.setShowPlayer(player);` and `.show(player, 0);` methods with the coccect timing.
2. Or use BetonQuest, that is the plugin i develop, to reproduce this issue, but it needs some more knowledge to setup. If you want to reproduce it with BetonQuest, i can give further details for this.

### Solution

Normally BetonQuest call the methods correctly in order. But as the visibility state is updated in a separate thread this is not guaranteed on a reload, when we need to destroy the old holograms and create new ones.

I thought that the enabled boolean could be changed into a AtomicBoolean to fix this issue, but it had no effect. This is my not working solution: https://github.com/DecentSoftware-eu/DecentHolograms/compare/main...Wolf2323:DecentHolograms:concurrency

### Server Version

git-Paper-296 (MC 1.19.2)

### Client Version

1.19.3

### Plugin Version

2.7.9

### Log

https://pastebin.com/VnDXJYHJ

**Comment by Andre601** [comment]
Pretty sure this isn't a bug and just the client not accepting the packets in time to remove the item (or a race-condition where some packets cancel out others?)

Either way, if you rejoin or switch worlds should this issue fix itself. And under normal circumstances would you not refresh or reload items that frequently.

tl;dr: Not a DH issue, but Minecraft being stupid.

**Comment by Wolf2323** [comment]
Ok, i am not 100% sure if i can confirm that this is an issue on my end only, but it seems to be, and i close this now. If i have news on this i will comment them here too.

**Comment by Wolf2323** [comment]
Small update on this, it was a concurrency issue on out end, but i had to change a lot of code to fix this (multiple factors causing this issue)
